src/main/java/de/fraunhofer/aisec/cpg/frontends/cpp/CXXLanguageFrontend.java

```diff
@@ -404,8 +415,8 @@ public void processAttributes(@NonNull Node node, @NonNull IASTAttributeOwner ow
     return list;
   }
 
-  private List<Expression> handleTokenList(IASTTokenList tokenList) {
-    List<Expression> list = new ArrayList<>();
+  private List<AnnotationMember> handleTokenList(IASTTokenList tokenList) {
+    List<AnnotationMember> list = new ArrayList<>();
 
     for (IASTToken token : tokenList.getTokens()) {
       if (token.getTokenType() == 6) {
```

Comment:
Is there a constant for the token type that can be used for comparison or could we comment what the token type 6 means?

Reply:
No, there isn't.

Reply:
I have added comments